MonoGame.Framework.Content.Pipeline/OpenAssetImporter.cs

```diff
@@ -198,7 +238,7 @@ public override NodeContent Import(string filename, ContentImporterContext conte
 			catch { }
 #endif
 
-            _identity = new ContentIdentity(filename, string.IsNullOrEmpty(ImporterName) ? GetType().Name : ImporterName);
+            _identity = new ContentIdentity(filename);
```

Comment:
@cra0zy Yes, but what why remove it? For debugging it can be helpful to know which importer ("FbxImporter", "CustomFbxImporter", etc.) created the content.